tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -158,6 +158,9 @@ public function testSendContextSsl()
             'ssl_verify_peer' => true,
             'ssl_verify_depth' => 9000,
             'ssl_allow_self_signed' => false,
+            'proxy' => [
+                'proxy' => '127.0.0.1:8080'
```

Comment:
Maybe we should also cover all parameters of stream context?

On Mon, Aug 31, 2015 at 7:02 PM, Mark Story <notifications@github.com>
wrote:

>> @@ -158,6 +158,9 @@ public function testSendContextSsl()
>>              'ssl_verify_peer' => true,
>>              'ssl_verify_depth' => 9000,
>>              'ssl_allow_self_signed' => false,
>> +            'proxy' => [
>> +                'proxy' => '127.0.0.1:8080'
> Ok, I forgot about the user/password :blush: 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cakephp/cakephp/pull/7326/files#r38333038